### PR TITLE
Team/team collection setup tt

### PR DESCRIPTION
**Pull request by tylertomlinson**
Team and Team Collection

### CODE REVIEW

**Review by d-atkins — APPROVED** [review]
I don't see any issues.

File: test/team_test.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+require './test/test_helper'
+require './lib/team'
+
+class TeamTest < Minitest::Test
+
+  def setup
+    @team= Team.new({team_id: 4, team_name: "Barcelona"})
```
Comment (mronauli) [inline test/team_test.rb:7]: probably not that big of an issue but maybe add a space after @team

File: lib/team_collection.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    load_from_csv(file_path, Team)
+  end
+
+  def number_of_teams
+    @teams_array.length
+  end 
+
+  def team_name_by_id(team_id)
+    @teams_array.find {|team| team.team_id == team_id}.team_name
+  end
```
Comment (mronauli) [inline lib/team_collection.rb:23]: awesome!

File: lib/team.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class Team
+
+attr_reader  :franchise_id, :team_name, :abbreviation, :link
```
Comment (Yetidancer) [inline lib/team.rb]: remains to be seen whether we will need all these instance variables

File: lib/team.rb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+class Team
+
+attr_reader :team_name, :team_id
+
+  def initialize(team)
+    @team_name = team[:teamname]
```
Comment (mronauli) [inline lib/team.rb:6]: underscore in :team_name

File: lib/team_collection.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+class TeamCollection
+  include CsvLoadable
+
+  attr_reader :teams_array
+
+  def initialize(file_path)
+    @teams_array = create_teams_array(file_path)
+  end
+
+  def create_teams_array(file_path)
```
Comment (Yetidancer) [inline lib/team_collection.rb:13]: word

File: lib/team.rb (hunk 3 above)
Comment (tylertomlinson) [reply lib/team.rb]: Agreed

File: lib/team.rb (hunk 4 above)
Comment (tylertomlinson) [reply lib/team.rb:6]: That is how it get the data from the CSV. Took a long time to figure out that was the case. Good catch though!

**Comment by Yetidancer** [comment]
let er rip

**Pull request merged**